assets/javascript/game.js

```diff
@@ -48,55 +67,54 @@ function createButtons() {
         letterBtn.classList.add('btn-outline-info');
         letterBtn.innerHTML = currentLetter;
         letterBtn.value = currentLetter;
-        letterBtn.addEventListener("click", guessLetter);
+        letterBtn.addEventListener('click', guessLetter);
         btnContainer.appendChild(letterBtn);
     }
 }
 
-createButtons();
-
-
-//Create the onClick functions for the letters
+// Create the onClick functions for the letters
 function guessLetter(ev) {
     console.log(this.value);
     this.setAttribute('disabled', true);
-    //If value returns true
+    // If value returns true
     if (answerLetters[this.value]) {
         console.log('right');
         answerLetters[this.value] = false;
-        var letterSpans = document.querySelectorAll('[name="'+this.value+'"]');
+        var letterSpans = document.querySelectorAll('[name="' + this.value + '"]'); // Make sure you research querySelector/querySelectorAll so you can understand what they can be used for.
         for (var i = 0; i < letterSpans.length; i++) {
             letterSpans[i].classList.remove('hide-letter');
         }
-    }
-    //If value returns false
-    else {
+    } else {
+        //If value returns false
         console.log('wrong');
         guessWrong++;
     }
     checkGameStatus();
 }
 
-//Determine if game is over
+// Determine if game is over
 function checkGameStatus() {
     var gameWon = true;
     var answerKeys = Object.keys(answerLetters);
     for (var i = 0; i < answerKeys.length; i++) {
         if (answerLetters[answerKeys[i]]) {
-            gameWon = false; 
+            gameWon = false;
             break;
         }
     }
-    //I word is completed
+    // I word is completed
     if (gameWon) {
         var winner = document.getElementById('winner');
-            winner.style.display= "block";
-    }
-    //If user runs out of guess
-    else if (guessWrong == maxGuesses) {
+        winner.style.display = 'block';
+    } else if (guessWrong == maxGuesses) {
+        // If user runs out of guess
         var loser = document.getElementById('loser');
-            loser.style.display= "block";
+        loser.style.display = 'block';
     }
     var gameLost = true;
 }
-    
+
+// I moved these funtion calls down here as it is usually advisable to define all functions before you start calling any of them
+// In this case it doesn't change the functionality, but in some cases it can, so usually you call any funtions you need to after ALL funtion are defined.
+createButtons();
```

Comment:
here is where I moved the function calls to create the buttons and initialize the game to. read my comment in the file to understand why.